### DISCUSSION TITLE
V2.2.17 spell slot cost is always 0

### DISCUSSION THREAD

**Discussion by Stadhouder** [misthero/dnd5e-spellpoints#111] [Q&A]
As I'm still using D&D version 3.3.1 I can't update beyond V2.2.17. Yet casting spells no longer consumes spell points. No matter what I change, the point cost is always 0. This persists even with refreshing the Spell Points item and with all other modules disabled.

**Comment by misthero** [comment]
are you on Foundry v11 or v12?

**Comment by Stadhouder** [comment]
V12 Build 331

**Comment by misthero** [comment]
so you need version 2.2.33.
you can find it here: https://github.com/misthero/dnd5e# -spellpoints/releases/tag/v2.2.33

remove your version and install that one using the manifest in the link.

after that lock the module to prevent auto-update (right click the module in foundry modules page and choose "lock") until you decide to upgrade your dnd installation

**Comment by Stadhouder** [comment]
That fixed it! Thank you so much!
